src/join.ts

```diff
@@ -95,35 +100,125 @@ function parseCallConfig(callConfig: string): DailyCallOptions {
   }
 }
 
+// getModifySdpHook takes a desired codec and returns the given hook to pass
+// to Daily to prefer that codec.
+function getModifySdpHook(wantedCodec: string): ((rtcSDP: any) => any) | null {
+  if (wantedCodec === '') {
+    return null;
+  }
+  const valid = ['VP8', 'VP9', 'H264'];
+  const codecName = wantedCodec.toUpperCase();
+  let found = false;
+  for (let i = 0; i < valid.length; i += 1) {
```

Comment:
Does this work...
```typescript
const found = valid.includes(codecName);
```
?

Reply:
I think it does! Tbh my instinct is to stay away from abstracted away iterations but in this case it might be a bit overkill xD I'll change it to use `includes`